dbms/tests/queries/0_stateless/01018_ddl_dictionaries_create.sql

```diff
@@ -0,0 +1,213 @@
+SET send_logs_level = 'none';
+
+DROP DATABASE IF EXISTS database_for_dict;
+
+CREATE DATABASE database_for_dict Engine = Ordinary;
+
+DROP TABLE IF EXISTS database_for_dict.table_for_dict;
+
+CREATE TABLE database_for_dict.table_for_dict
+(
+  key_column UInt64,
+  second_column UInt8,
+  third_column String
+)
+ENGINE = MergeTree()
+ORDER BY key_column;
+
+INSERT INTO database_for_dict.table_for_dict VALUES (1, 100, 'Hello world');
+
+DROP DATABASE IF EXISTS ordinary_db;
+
+CREATE DATABASE ordinary_db ENGINE = Ordinary;
+
+SELECT '=DICTIONARY in Ordinary DB';
+
+DROP DICTIONARY IF EXISTS ordinary_db.dict1;
+
+CREATE DICTIONARY ordinary_db.dict1
+(
+  key_column UInt64 DEFAULT 0,
+  second_column UInt8 DEFAULT 1,
+  third_column String DEFAULT 'qqq'
+)
+PRIMARY KEY key_column
+SOURCE(CLICKHOUSE(HOST 'localhost' PORT 9000 USER 'default' TABLE 'table_for_dict' PASSWORD '' DB 'database_for_dict'))
+LIFETIME(MIN 1 MAX 10)
```

Comment:
As I can see - we have here two mandatory options (min, max). Maybe, it is better to omit them, allowing to specify in a form like ```LIFETIME(1,10)```?
Pros:
1. No extra syntax constructions needed
2. Less symbols to type :)
3. Looks more native for SQL users (like function call)

Cons:
1. If lifetime will have more complicated configuration, it will get stuck. E.g.: ```LIFETIME(MAX 10 IF_ONCE_QUERIED_REMEMBER_FOREVER 1)``` (here MIN 1 is omitted, but another option is introduced)

Reply:
Current `LIFETIME` definition is consistent with other parts `RANGE`, `SOURCE` , etc.